src/transformers/models/vit/convert_vit_timm_to_pytorch.py

```diff
@@ -0,0 +1,1271 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Convert ViT checkpoints from the timm library."""
+
+
+import argparse
+from pathlib import Path
+
+import torch
+import torchvision.transforms as T
+from PIL import Image
+
+import requests
+import timm
+from transformers import ViTConfig, ViTFeatureExtractor, ViTForImageClassification
+from transformers.utils import logging
+
+
+logging.set_verbosity_info()
+logger = logging.get_logger(__name__)
+
+# ImageNet 2012 id's to class names
+id2class = {
```

Comment:
Can we store this in a file in transformers.utils instead (so you would import it from there)? This might be needed for other models in the future.

Reply:
I've stored it in a file in transformers.utils named `imagenet_classes.py`. Let me know if this looks ok to you.